examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -43,7 +43,7 @@ data:
       customServiceConfig: |
         [DEFAULT]
         debug = true
-        enabled_share_protocols=cifs
+        enabled_share_protocols = nfs
```

Comment:
This updates beta a similar change was made to delta below but `beta =! delta`

https://github.com/openstack-k8s-operators/architecture/pull/282/files